package.json

```diff
@@ -12,8 +12,22 @@
     "@next/font": "13.1.3",
     "eslint": "8.32.0",
     "eslint-config-next": "13.1.3",
+    "lottie-react": "^2.3.1",
     "next": "13.1.3",
     "react": "18.2.0",
-    "react-dom": "18.2.0"
+    "react-awesome-reveal": "^4.2.3",
+    "react-dom": "18.2.0",
+    "react-easy-emoji": "^1.8.1",
+    "react-lottie": "^1.2.3",
```

Comment:
I think we can remove react-lottie and react-reveal since they are no longer maintained and incompatible with react 18. The react-awesome-reveal and lottie-react packages seem to be working fine as a replacement.

Reply:
Yep if that's the case then you can remove it